.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
               uses: ballerina-platform/ballerina-action/@master
```

Comment:
@LakshanSS , Do we need to have the `@master` as the ballerina action

Reply:
Now we'll keep it like that.
After SLBeta4 is released, if we want to do a release in slbeta4, we need to change this as slbeta4
Or After SwanLake GA release is done, we'll change this to relevant version.